index.js

```diff
@@ -106,9 +106,17 @@ module.exports._flush = function(lastTime, code) {
 
   return RSVP.Promise.resolve(lastTime).
     then(function() {
-      return RSVP.Promise.all(work.map(function(handler) {
+      return RSVP.allSettled(work.map(function(handler) {
         return handler.call(null, code);
-      }));
+      })).then(function(results) {
+        var firstRejectedHandler = results.find(function(p) {
```

Comment:
We can probably remove support for it ([see here](https://github.com/ember-cli/ember-cli/pull/6622)), but it should probably be a "major version" bump (i.e. v2.0 here).

My suggestion would be to use `.filter(currentFunction)[0]` for now (so that we can get this fix released in v1.x) then replace (or open an issue so we don't forget) and release v2.x.